index.d.ts

```diff
@@ -624,7 +624,7 @@ declare module 'binance-api-node' {
     }): Promise<AggregatedTrade[]>
     futuresTrades(options: { symbol: string; limit?: number }): Promise<TradeResult[]>
     futuresUserTrades(options: {
-      symbol: string
+      symbol?: string
```

Comment:
is the symbol really optional?

Reply:
Yes. While I am making my bot, and checking the docs I am making changes to my local node_modules folder and them, I am sharing this changes here. 

My sugestions are proved with real money before share in PR's.